src/autospawn/auto_spawn.ts

```diff
@@ -0,0 +1,76 @@
+
+export class AutoSpawn {
+    static CONSONANTS = <string>"bcdfghjklmnpqrstvwxz"
+
+    static VOWELS = <string>"aeiouy"
```

Comment:
Typescript will figure out that these types are strings without explicitly having the `<string>` thing there.